packages/node-resolve/README.md

```diff
@@ -75,14 +75,7 @@ If `true`, instructs the plugin to use the browser module resolutions in `packag
 Type: `Array[...String]`<br>
 Default: `['node_modules']`
 
-A list of directory names in which to recursively look for modules.
-
-### `modulePaths`
-
-Type: `Array[...String]`<br>
-Default: `[]`
-
-A list of absolute paths to additional locations to search for modules. [This is analogous to setting the `NODE_PATH` environment variable for node](https://nodejs.org/api/modules.html#loading-from-the-global-folders).
+One or more directories in which to recursively look for modules.
```

Comment:
I don't see this change mentioned in the PR description or the linked issues. Is there a reason that the `modulePaths` option was removed?

Reply:
Similar: I didn't deliberately do this. I can't figure why either the GitHub editor or Webstorm would have made these changes?

Ugh. Too late for me today, I'll fix it tomorrow.